libs/application/templates/estate/src/fields/EstateMembersRepeater/AdditionalEstateMember.tsx

```diff
@@ -258,7 +261,7 @@ export const AdditionalEstateMember = ({
                       alertWhenUnder18:
                         selectedEstate !==
                           EstateTypes.divisionOfEstateByHeirs &&
-                        kennitala.info(currentEstateMember.nationalId).age < 18,
+                          memberAge !== undefined && memberAge < 18,
```

Comment:
same here

Reply:
!!memberAge would return false for 0 though